### PR TITLE
Improve common service models and service definitions

### PR DESCRIPTION
**Pull request by MSurmach**

### CODE REVIEW

File: src/main/proto/user.proto [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+message UserDto {
+  uint64 id = 1;
+  string first_name = 2;
+  string last_name = 3;
+  string email = 4;
+  string phone_number = 5;
+}
+
+message UserIdDto {
+  uint64 id = 1;
```
Comment (svetamint) [inline src/main/proto/user.proto:19]: I'd suggest rename `UserIdDto` to `GetUserRequest` like as `CreateUserRequest` etc

File: src/main/proto/user.proto [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+message UserDto {
+  uint64 id = 1;
+  string first_name = 2;
+  string last_name = 3;
+  string email = 4;
+  string phone_number = 5;
+}
+
+message UserIdDto {
```
Comment (svetamint) [inline src/main/proto/user.proto:18]: I'd suggest rename `UserIdDto` to `GetUserRequest`

File: src/main/proto/user.proto [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+message FindUserByEmailRequest {
+  string email = 1;
+}
+
+message FindAllResponse {
+  repeated UserDto users = 1;
+}
+
+service UserService {
+  rpc Create(CreateUserRequest) returns (UserDto);
```
Comment (svetamint) [inline src/main/proto/user.proto:42]: please, follow Java naming conventions - all method names should be started with the first lowercase letter

File: src/main/proto/user.proto [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+message UpdateUserRequest {
+  UserDto user_dto = 1;
+}
+
+message FindUserByEmailRequest {
+  string email = 1;
+}
+
+message FindAllResponse {
+  repeated UserDto users = 1;
```
[the remark this answers is not in the document]
Comment (MSurmach) [reply src/main/proto/user.proto]: Reworked all message and service names

File: src/main/proto/parcel.proto [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  string destination = 5;
+  uint64 user_id = 6;
+}
+
+message UpdateTrackingStatusRequest {
+  string tracking_number = 1;
+  TrackingStatus new_status = 2;
+}
+
+message FindAllResponse {
```
[the remark this answers is not in the document]
Comment (MSurmach) [reply src/main/proto/parcel.proto]: Reworked all message and service names

File: src/main/proto/user.proto (hunk 2 above)
Comment (MSurmach) [reply src/main/proto/user.proto]: Reworked all message and service names

File: src/main/proto/user.proto (hunk 3 above)
Comment (MSurmach) [reply src/main/proto/user.proto]: Reworked all message and service names

**Review by svetamint — APPROVED** [review]

**Pull request merged**